plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalogFactory.java

```diff
@@ -78,6 +79,7 @@ public TrinoGlueCatalogFactory(
         this.hideMaterializedViewStorageTable = icebergConfig.isHideMaterializedViewStorageTable();
         this.stats = requireNonNull(stats, "stats is null");
         this.isUsingSystemSecurity = securityConfig.getSecuritySystem() == SYSTEM;
+        this.perQueryManifestCacheSizeBytes = icebergConfig.getPerQueryManifestCacheSize().toBytes();
```

Comment:
Is `TrinoIcebergRestCatalogFactory` left out of the PR intentionally? If yes, pls document this "limitation" in the PR description